README-CHANGES.xml

```diff
@@ -89,6 +89,7 @@
           </c:tickets>
         </c:change>
         <c:change date="2021-09-28T00:00:00+00:00" summary="Fix long chapter titles overlapping elapsed/total time displays in audiobook player"/>
+        <c:change date="2021-09-30T11:37:51+00:00" summary="Added support to featured libraries"/>
```

Comment:
Should be "Added support _for_ featured libraries". "Support to" sounds a bit weird here.

Reply:
Non-native english mistake, sorry :D